_includes/footer.html

```diff
@@ -0,0 +1,9 @@
+<div class="usa-grid">
+  <div class="usa-site-footer-content usa-width-one-half">
```

Comment:
I guess I don't understand the need to have a seperate bit of CSS for the style guide (perhaps beyond doing some margin fixing for presentation inside another grid). Ultimately, I'm going to be looking through the source of the documentation site to determine the class structure for elements and I don't think I'm going to remember that distinction all the time.

When I look at other CSS frameworks they generally use the framework itself for the documentation.